install/saaj/bin/ts.jte

```diff
@@ -68,7 +68,7 @@ endorsed.dirs=
 ####################################################################
 ts.harness.classpath=${ts.home}/lib/javatest.jar${pathsep}${ts.home}/lib/tsharness.jar${pathsep}${ts.home}/lib/saajtck.jar${pathsep}${ant.home}/lib/ant.jar${pathsep}${ts.home}/lib/jaxb-api.jar${pathsep}${ts.home}/lib/jaxb-xjc.jar${pathsep}${ts.home}/lib/jaxb-libs.jar${pathsep}${ts.home}/lib/jaxb-impl.jar
 
-ts.classpath=${local.classes}${pathsep}${ts.home}/lib/tsharness.jar${pathsep}${ts.home}/lib/saajtck.jar
+ts.classpath=${local.classes}${pathsep}${ts.home}/lib/tsharness.jar${pathsep}${ts.home}/lib/saajtck.jar${pathsep}${ts.home}/lib/jakartaee-api.jar
```

Comment:
For the TCKs, the API classes are better suited in `local.classes`

Reply:
This pr needs to update install.saaj.bin/ts.jte, to update local.classes, instead of ts.classpath, as per @jansupol advice.

Reply:
I'd like to, but it does not work.